pkg/registry/apis/iam/resourcepermission/queries/role_insert.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO {{ .Ident .RoleTable }} (version, org_id, uid, name, created, updated)
+VALUES (
+    0,
```

Comment:
does it default to 0?

Reply:
I had to add it otherwise it failed the query.